### PR TITLE
postinst: Adapt script for new messages module

### PR DESCRIPTION
**Pull request by lucasmoura**
postinst: Adapt script for new messages module

## Test Steps
Run any of the PRO FIPS tests we have and assert that no package related issues are raised

## Desired commit type

 - [ ] Squash and merge: Using the "Proposed Commit Message"
 - [ ] Create a merge commit and use "Proposed Commit Message"
 - [x] Rebase and merge, no merge commit, rely only on existing commit messages

## Checklist

 - [ ] I have updated or added any unit tests accordingly
 - [ ] I have updated or added any integration tests accordingly
 - [ ] I have updated or added any documentation accordingly

### CODE REVIEW

File: uaclient/messages.py [hunk 1]
```diff
@@ -102,6 +102,7 @@ class TxtColor:
 LIVEPATCH_LTS_REBOOT_REQUIRED = (
     "Livepatch support requires a system reboot across LTS upgrade."
 )
+FIPS_REBOOT_REQUIRED_MSG = "Reboot to FIPS kernel required"
```
Comment (renanrodrigo) [inline uaclient/messages.py:105]: I guess `FIPS_REBOOT_REQUIRED` should be enough

**Comment by CalvoM** [comment]
Looks good to me, The change talks it well. Just a quick question, why do we have some messages in messages.py and some in status.py?

**Review by renanrodrigo — APPROVED** [review]
lgtm;
Is there a reason to keep that set of messages (Notices, Prompts, etc) in the status module, instead of moving them to messages? Not for this PR I guess

**Comment by renanrodrigo** [comment]
@CalvoM was 25 seconds faster than me

File: uaclient/messages.py (hunk 1 above)
Comment (lucasmoura) [reply uaclient/messages.py:105]: It should, but we have a `NamedMessage` with this name already and I don't want to further expand on this at this moment

**Comment by lucasmoura** [comment]
@CalvoM the rationale is that I believe we should have a dedicated `notices` module in the future. I could move all of the notices into the `messages` module, but that would trigger more refactoring of the code. That's why I am not changing the notices in this PR

File: uaclient/messages.py (hunk 1 above)
Comment (renanrodrigo) [reply uaclient/messages.py:105]: makes all sense

**Comment by renanrodrigo** [comment]
So yes, I think prompts and notices are, before anything else, messages - and thus could live together in the messages module, as they were in status for instance, without any downside.
Also agreed that this PR now does not have to cover that.
:shipit:

**Pull request merged**